opensrp-chw/src/miecd/java/org/smartregister/chw/interactor/ChildHomeVisitInteractorFlv.java

```diff
@@ -0,0 +1,270 @@
+package org.smartregister.chw.interactor;
+
+import android.text.TextUtils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.LocalDate;
+import org.joda.time.format.DateTimeFormat;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.smartregister.chw.R;
+import org.smartregister.chw.anc.actionhelper.HomeVisitActionHelper;
+import org.smartregister.chw.anc.model.BaseAncHomeVisitAction;
+import org.smartregister.chw.util.Constants;
+import org.smartregister.chw.util.JsonFormUtils;
+import org.smartregister.immunization.domain.ServiceWrapper;
+
+import java.text.MessageFormat;
+import java.util.Map;
+
+import timber.log.Timber;
+
+public class ChildHomeVisitInteractorFlv extends DefaultChildHomeVisitInteractorFlv {
+
+    @Override
+    protected void bindEvents(Map<String, ServiceWrapper> serviceWrapperMap) throws BaseAncHomeVisitAction.ValidationException {
+        try {
+            evaluateImmunization();
+            evaluateExclusiveBreastFeeding(serviceWrapperMap);
+            evaluateVitaminA(serviceWrapperMap);
+            evaluateDeworming(serviceWrapperMap);
+            evaluateMalariaPrevention();
+            evaluateCounselling();
+            evaluateNutritionStatus();
+            evaluateObsAndIllness();
+        } catch (BaseAncHomeVisitAction.ValidationException e) {
+            throw (e);
+        } catch (Exception e) {
+            Timber.e(e);
+        }
+    }
+
+    protected void evaluateImmunization() throws Exception {
+        setVaccinesDefaultChecked(false);
+        super.evaluateImmunization();
+    }
+
+    private void evaluateMalariaPrevention() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'evaluateMalariaPrevention()' has an NPath complexity of 1152](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426631)